### DISCUSSION TITLE
p/0f2182e4368c4195be5f7699fa62e026/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#9] [Announcements]
# p/0f2182e4368c4195be5f7699fa62e026/

前言一些用不到的东西想出掉，所以在这里发一下。  走闲鱼，有意向请使用 mail/telegram 联系，吾辈会上咸鱼挂上去发链接 不讲价，浪费的时间可能很容易超过物品的价值 不包邮，所以如果是书的话最好不要只买一本  电子垃圾[ ] Wacom 数位板 CTL672好吧，之前想替代纸笔处理流程图之类的事情，后来证明是吾辈想多了。300 出   淘宝链接: https://detail.tmall

https://blog.rxliuli.com/p/0f2182e4368c4195be5f7699fa62e026/

**Comment by ryan4yin** [comment]
这么用心一本本拍上来，不过想全部出掉的话，感觉只能找多抓鱼之类的二手书店

**Reply by rxliuli** [reply]
最晚明年下半年吾辈出去的时候肯定要出掉，不然就只能寄给朋友了（其实还有 kigurumi 之类的也很头疼，毕竟出去之后大抵只能带一个行李箱+一个包，其他东西要么寄回家里，要么出掉，要么只能扔了（但老实说不想扔书

**Reply by ryan4yin** [reply]
rxliuli 明年有润霓虹的打算？

书其实还挺好卖的，我几年前毕业时就在多抓鱼上卖了不少。
不过还有很多不想卖的书，我打包寄回家了，现在在老家摆着。

kigurumi 这个感觉就没啥好办法了，寄回家显然不太合适...

**Reply by ryan4yin** [reply]
多抓鱼是一个微信小程序，直接扫书的 ISBN 码录入，就能提示预计价格，然后会安排顺丰到付上门取货，记得是满 80 包邮，很方便的。
